src/Config/SystemConfigReader.ts

```diff
@@ -0,0 +1,153 @@
+import { logger } from '../Utils';
+import request, { RequestPromiseAPI } from 'request-promise';
+import { promises, constants } from 'fs';
+import { join } from 'path';
+import { homedir } from 'os';
+
+const { readFile, writeFile, access } = promises;
```

Comment:
In fact, we are still supporting Node.js 10, as stated here: https://github.com/NeuraLegion/bright-cli/blob/19f53a66c687ec8ca37ee0f70dbea1d3d32b4c61/package.json#L25. Please refrain from using capabilities introduced in later versions. Use the `promisify` instead

Reply:
According to [the docs](https://nodejs.org/docs/latest-v10.x/api/fs.html#fs_fs_promises_api), fs.promises were introduced in Node.js 10.0.0. as [experimental feature](https://nodejs.org/en/blog/release/v10.0.0).

The first 10.x LTS was `10.13.0` where they stayed experimental, and in 10.17.0 it became [stable](https://github.com/nodejs/node/pull/29875).

So if our purpose is to support only latest LTS version, then `fs.promises` will work fine on 10.x LTS. Otherwise it will still work, but users will see `Experimental feature` warning in their console.

Reply:
> So if our purpose is to support only latest LTS version

This is the aim, as mentioned earlier, we continue to provide support for a wide range of versions, starting from v10.x.x due to the constrain in package.json.

> Otherwise it will still work, but users will see Experimental feature warning in their console.

This is unacceptable from both the user and developer perspectives (as it's the only single place where this module is utilized).